### PR TITLE
folder structure and funcntion done

### PR DESCRIPTION
**Pull request by mubeenfayyaz20**

### CODE REVIEW

File: src/pages/Dashboard/ChildComponent.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const ChildComponent = ({ heading, button, onMouseHover }) => {
```
Comment (ahsan-bilal-786) [inline src/pages/Dashboard/ChildComponent.jsx]: Please rename this component with a proper well-defined name?

File: src/pages/Dashboard/ChildComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  return (
+    <div
+      className="childComponent"
+      style={{ border: "1px solid red", padding: "10px", borderRadius: "10px" }}
```
Comment (ahsan-bilal-786) [inline src/pages/Dashboard/ChildComponent.jsx]: avoid inline styling . implement the styling in styled component.

File: src/pages/Dashboard/ParentComponent.jsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import ChildComponent from "./ChildComponent";
+
+const ParentComponent = () => {
```
Comment (ahsan-bilal-786) [inline src/pages/Dashboard/ParentComponent.jsx]: Please rename it to well define which should be helpful to understand the objective of this code or component.

**Review by ahsan-bilal-786 — CHANGES_REQUESTED** [review]

File: src/pages/Dashboard/ChildComponent.jsx (hunk 2 above)
Comment (mubeenfayyaz20) [reply src/pages/Dashboard/ChildComponent.jsx]: done

File: src/pages/Dashboard/ParentComponent.jsx (hunk 3 above)
Comment (mubeenfayyaz20) [reply src/pages/Dashboard/ParentComponent.jsx]: done

File: src/pages/Dashboard/ChildComponent.jsx (hunk 1 above)
Comment (mubeenfayyaz20) [reply src/pages/Dashboard/ChildComponent.jsx]: done

**Review by ahsanbilalas — APPROVED** [review]

**Pull request merged**